src/app-layout/drawer/interfaces.ts

```diff
@@ -19,26 +20,26 @@ export interface DesktopDrawerProps {
   width: number;
   topOffset: number | undefined;
   bottomOffset: number | undefined;
-  ariaLabels: AppLayoutProps.Labels | undefined;
-  drawersAriaLabels?: { mainLabel: string | undefined; closeLabel: string | undefined; openLabel: string | undefined };
+  ariaLabels: {
+    mainLabel: string | undefined;
+    closeLabel: string | undefined;
```

Comment:
any reason why we're not using our more common strategy of marking optional props with `?` ?

Reply:
This is important when passing the props down

```
<Drawer ariaLabels={{}}> // TS error that the object does not have the required keys

<Drawer ariaLabels={{ mainLabel: props.ariaLabels.mainLabel }}> // TS passes, even if the property eventually can be undefined too
```

Optional `prop?: string` is good for public API, but for all internal components, we mostly want `prop: string | undefined`